ARK Server Manager/ServerSettingsControl.xaml.cs

```diff
@@ -163,11 +163,12 @@ private void ReinitializeNetworkAdapters()
             preferredIP.Description = "(Recommended) " + preferredIP.Description;
             if(String.IsNullOrWhiteSpace(this.Settings.ServerIP))
             {
-                if(preferredIP != null)
-                {
-                    this.Settings.ServerIP = preferredIP.IPAddress;
-                }
-            } 
+                // removed to enforce the 'Let ARK choose' option.
```

Comment:
There is already an option in the drop list for Let Ark Choose. When it is selected it clears out the multihome values in the ini files.
The problem is that when this option is selected and ASM is closed and reopened, the code I commented out, resets the multihome back to the recommended IP Address and does not leave it blank.